crates/sui-core/src/authority/authority_store.rs

```diff
@@ -1612,9 +1604,13 @@ impl AuthorityStore {
                 .simplified_unwrap_then_delete(),
         );
 
-        let root_state_hash = self
-            .get_root_state_hash(cur_epoch_store.epoch())
-            .expect("Retrieving root state hash cannot fail");
+        let root_state_hash: ECMHLiveObjectSetDigest = self
```

Comment:
This entire method can be removed? Looks like it is replaced by expensive_check_is_consistent_state in AuthorityState

Reply:
yes, thanks, i meant to do that